lib/sycamore/sycamore/llms/llms.py

```diff
@@ -17,6 +17,9 @@ def generate(self, *, prompt_kwargs: dict, llm_kwargs: Optional[dict] = None) ->
     def is_chat_mode(self) -> bool:
         pass
 
+    async def generate_future(self, *, prompt_kwargs: dict, llm_kwargs: Optional[dict] = None) -> Awaitable[str]:
```

Comment:
We should call this generate_async or generate_awaitable since it's not returning specifically a future.

Reply:
fixed!